core/arch/arm/kernel/generic_entry_a32.S

```diff
@@ -34,6 +34,37 @@
 #include <sm/teesmc_opteed_macros.h>
 #include <sm/teesmc_opteed.h>
 
+.section .data
+.balign 4
+
+#ifdef CFG_BOOT_SYNC_CPU
+/*
+ * Semaphore to let primary CPU runs its inits
+ * before secondary CPUs run their own inits
+ */
+/* Semaphores sync CPU0 and CPU1 */
+.equ SEM_NOT_READY, 2
+.equ SEM_CPU_READY, 4
+
+sem_cpu_sync:
+	.word SEM_NOT_READY;	/* cpu0 */
```

Comment:
This can probably be allocated with some macro trickery and `CFG_TEE_CORE_NB_CORE`, if not we can use the technique used in thread.c for `stack_tmp_top`.